lms/djangoapps/shoppingcart/models.py

```diff
@@ -1545,12 +1636,14 @@ def contained_in_order(cls, order, course_id):
         ]
 
     @classmethod
-    def get_total_amount_of_purchased_item(cls, course_key):
+    def get_total_amount_of_purchased_item(cls, course_key, status='purchased'):
```

Comment:
Where are the tests? Ideally there should be a test_models.py with a `PaidCourseRegistrationTests` class and `test_*` methods for each of these new methods. I'm not seeing these methods referenced in any test files.